src/imagery/i.eodag/i.eodag.html

```diff
@@ -52,7 +52,8 @@ <h4>Searching using scenes IDs</h4>
 Users have a set of scenes IDs that they want to search for and download.
 They can either use the <b>id</b> option or use the <b>file</b> option
 and pass a text file, with one ID per line.
-In both cases, specifying a provider is optional. In case users do
+When searching with ID, user must specify the product type that the IDs belong to.
+Specifying a provider is optional. In case users do
 not specify the provider, products might be offered
 from different providers as long as the user provides the
 credentials in the configuration file.
```

Comment:
```suggestion
credentials in the configuration file and the IDs are the same among providers.
```

Reply:
I updated this part here https://github.com/OSGeo/grass-addons/pull/1165/commits/dae18f367b6af74f6d5942c902e51d5f857e1775

I am trying to say that, if a provider is not specified then each scene could be offered using a different provider, but each scene will only be offered once, using one provider.